doc/reline/face.md

```diff
@@ -0,0 +1,108 @@
+# Face
+
+With the `Reline::Face` class, you can modify the text color and text decorations in your terminal emulator.
+This is primarily used to customize the appearance of the method completion dialog in IRB.
+
+## Usage
+
+### ex: Change the background color of the completion dialog cyan to blue
+
+```ruby
+Reline::Face.config(:completion_dialog) do |conf|
+  conf.define :default, foreground: :white, background: :blue
+  #                                                     ^^^^^ `:cyan` by default
+  conf.define :enhanced, foreground: :white, background: :magenta
+  conf.define :scrollbar, foreground: :white, background: :blue
+end
+```
+
+If you provide the above code to an IRB session in some way, you can apply the configuration.
+It's generally done by writing it in `.irbrc`.
+
+Regarding `.irbrc`, please refer to the following link: [[https://docs.ruby-lang.org/en/master/IRB.html]]
```

Comment:
Is `[[...]]` a special syntax for markdown? I don't see it being converted in rich diff 🤔

Reply:
It's my bad. You can use `[[url]]` in the _Sidebar.md in wiki page but not available here. fixed https://github.com/ruby/reline/pull/552/commits/1ce4818d41461af6a3c9b93fcb9d28cde13a1331